pages/docs/index.js

```diff
@@ -90,7 +90,7 @@ export default function DocsIndex() {
           name: "Bio",
           path: "/docs/how-to-guides/bio",
           description:
-            "The bio string in the json file is special! It allow for the use of Markdown. This will allow you to use the formatting styles like italic and bold.",
+            "The bio string in the json file is special! It allow for the use of Markdown. This will allows you to use the formatting styles like italic and bold.",
```

Comment:
this is supposed to be `allow`, `allows` is wrong here

Reply:
@Rishav-12 meant the allow after the It, `It allow for the use of Markdown` -> `It allows for the use of Markdown`